_SQLServer/TableLoadAbilities.sql

```diff
@@ -1,6 +1,193 @@
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Vending Machine', 'Action', 'Support', 2, -1, 6, 'Augment, Range 6. Remove a single potion counter from Candy''s card and place it on target Hero''s card.', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Soothing Soda', 'Potion', 'Support', -1, 1, -1, 'Immune: Status Effects.', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Sugar Rush', 'Potion', 'Support', 3, 1, -1, '+2ST', '1.0');--? Check values
+
+
+--Celestial Herald
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Arrow Storm', 'Action', 'Offense', 2, -1, 4, 'DEX vs ARM, Spray 4', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Heart Seeker', 'Action', 'Offense', 2, 1, 2, 'Missle 8, +1R DEX, Hookshot', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Halo Extract', 'Potion', 'Support', 0, -1, -1, 'Aura 1, +1ST ARM', '1.0');--? Check values
+
+
+--Claw Tribe Barbarian
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Rage', 'Action', 'Offense', 3, -1, -1, 'After every square you move this turn immediately make one melee attack.  You may not make a Berserk attack on a turn you Rage.  Barbarian''s ARM is reduced to 2B until her next activation.', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Fury''s Blood', 'Potion', 'Support', 0, -1, -1, '+1B ARM, Backlash', '1.0');--? Check values
+
+
+--Codifier Kisa
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Black Cat', 'Action', 'Offense', 2, -1, -1, 'Wave 1, Fragile', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Cat & Mouse', 'Action', 'Support', 1, -1, 6, 'Range 6, WILL vs WILL, Compel 3', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Scratch', 'Potion', 'Offense', 1, -1, -1, 'B Wave 2, Slow', '1.0');--? Check values
```

Comment:
Description should read 'Bac Wave 2, Slow'.
The card creator codewords don't account for a support action button, but "B" would result in a blue die.